<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefsMxp</class>
 <widget class="QWidget" name="PrefsMxp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MXP</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_title">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>MXP</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="_use_mxp">
     <item row="0" column="0">
      <widget class="QLabel" name="UseMxp_label">
       <property name="text">
        <string>Use MXP:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="UseMxp">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>On command</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>On query</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Always</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Never</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="IgnoreMxpColourChanges">
     <property name="text">
      <string>Ignore colour changes</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="SendMxpAfkResponse">
     <property name="text">
      <string>Send AFK response</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="MudCanChangeColorOptions">
     <property name="text">
      <string>MUD can change some options</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="_hyperlinks">
     <property name="title">
      <string>Hyperlinks</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="_custom_color">
        <item>
         <widget class="QCheckBox" name="UseCustomLinkColour">
          <property name="text">
           <string>Use custom link colour:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ColorPickerButton" name="HyperlinkColour">
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="MudCanChangeLinkColour">
        <property name="text">
         <string>MUD can change link colour</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="UnderlineHyperlinks">
        <property name="text">
         <string>Underline hyperlinks</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="MudCanRemoveUnderline">
        <property name="text">
         <string>MUD can remove underline</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="HyperlinkAddsToCommandHistory">
        <property name="text">
         <string>Save hyperlink mouse clicks in history</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="EchoHyperlinkInOutputWindow">
        <property name="text">
         <string>Echo hyperlink mouse clicks in output</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="reset_tags">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Reset tags</string>
     </property>
     <property name="icon">
      <iconset theme="edit-undo"/>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorPickerButton</class>
   <extends>QToolButton</extends>
   <header>colorpickerbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
